trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/BET_calculations.py

```diff
@@ -140,19 +140,11 @@ def compute_inflow_and_tip_loss(r,R,Wa,Wt,B):
     """
     lamdaw            = r*Wa/(R*Wt)
     lamdaw[lamdaw<0.] = 0.
-    f                 = (B/2.)*(1.-r/R)/lamdaw
-    f[f<0.]           = 0.
-    
-    piece             = np.exp(-f)
-    F                 = 2.*np.arccos(piece)/np.pi
 
-    Rtip = R
     et1, et2, et3, maxat = 1,1,1,-np.inf
-    tipfactor = B/2.0*(  (Rtip/r)**et1 - 1  )**et2/lamdaw**et3
+    tipfactor = B/2.0*(  (R/r)**et1 - 1  )**et2/lamdaw**et3 + 1e-3 # tolerance to prevent nan
```

Comment:
I ran into this issue before. [Check this out](https://github.com/suavecode/SUAVE/blob/jax-nexus_grad/trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/BET_calculations.py#L143).

Reply:
Modified